SingInTests/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SingInTests
+{
+    internal class SignInTest
+    {
+        private CreateUserRequest request = new CreateUserRequest
+        {
+            FirstName = "Test",
+            LastName = "Test",
+            Birthday = DateTime.Today.AddYears(-20),
+            Email = "testUser@mail.ru",
+            Password = "password"
+        };
+
+        private static void Main(string[] args)
+        {
+            var test = new SignInTest();
+            try
+            {
+                test.TryToSingUp();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("User was created");
+            }
+
+            if (test.TryToSingIn())
+            {
+                Console.WriteLine("Successful");
+            }
+            else
+            {
+                Console.WriteLine("Something was wrong");
+            }
+        }
+
+        private async Task TryToSingUp()
+        {
+
+            const string url = "https://localhost:5011/users/create";
+
+            var client = new RestClient<CreateUserRequest, bool>(url, RestRequestType.POST);
+
+            await client.Execute(request);
+        }
+
+        private bool TryToSingIn()
+        {
+            IWebDriver driver = new ChromeDriver();
```

Comment:
why not use var?

Reply:
I didn't use var, because I need this interface. This interface allows me to quickly replace the driver